tests/analyze_assoc.R

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/Rscript
+#
+# Copyright 2015 Google Inc. All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+library("getopt")
```

Comment:
Since we are already using "optparse" in tests/analyze.R, let's stick with that.  We're also using a similar optparse Python library everywhere now, so it's all consistent.